### DISCUSSION TITLE
This package is not supported anymore

### DISCUSSION THREAD

**Discussion by introwit** [JoggApp/laravel-google-translate#51] [Announcements]
We at Jogg no longer use this package hence we won't be actively developing it. We will be archiving this package permanently in the coming time. If you run and maintain a fork or a similar package feel free to link it below in this thread. ~Or if you want to maintain it, we can add you as a maintainer~. We will add it to our readme.

**Comment by boryn** [comment]
Hi @introwit!

We use this package on a regular basis. I could become the maintainer and care about the updates.


Cheers
Michal

**Comment by introwit** [comment]
@boryn hi, we have decided to just do away with the maintainer idea. Feel free to fork it, we can add a link to your fork in this packages readme.

**Comment by boryn** [comment]
Hi @introwit!

I have forked it to an organization: https://github.com/DynaSoft/laravel-google-translate
